### ISSUE TITLE
[SCREENREADER] GIBCT Section 116 - Font Awesome icons need aria-hidden attribute

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#3571]
## Description

There are a number of Font Awesome icons that are being picked up by JAWS as cursor stops, but do not announce themselves. These should have an `aria-hidden="true"` attribute to remove them from the cursor order. Screenshot attached.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a screen reader user, my virtual cursor should not stop on the icons on the search results or the provider detail view.

## Possible Fixes (optional)
Add an `aria-hidden` attribute to each icon. There are a number of them on multiple pages.

```html
<i class="fa fa-map-marker undefined" aria-hidden="true"></i>
```

## Screenshots or Trace Logs

![Screen Shot 2019-11-19 at 10.17.39 AM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/efb756c0-98f9-408e-9ec6-b21fe4cea1d2)

**Comment by hughes-dustin** [comment]
branch: bah-3571

**Comment by sonntag-adam** [comment]
Testing passes QA

**Comment by sonntag-adam** [comment]
@1Copenut change is in staging.

**Comment by 1Copenut** [comment]
@sonntag-adam I spot-checked with JAWS + IE11 on Staging. Verified the fix and am moving this ticket to close.
